src/test/java/org/minions/devfund/angela/katas/MorseCodeDecoderTest.java

```diff
@@ -0,0 +1,15 @@
+package org.minions.devfund.angela.katas;
+
+
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+
+public class MorseCodeDecoderTest {
+    @Test
+    public void testExampleFromDescription() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)